src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -32,6 +32,7 @@ static ErrorFacts()
             builder.Add(getId(ErrorCode.WRN_NullabilityMismatchInParameterTypeOfTargetDelegate));
             builder.Add(getId(ErrorCode.WRN_NullAsNonNullable));
             builder.Add(getId(ErrorCode.WRN_NoBestNullabilityConditionalExpression));
+            builder.Add(getId(ErrorCode.WRN_NullableValueTypeMayBeNull));
```

Comment:
>WRN_NullableValueTypeMayBeNull [](start = 40, length = 30)

not related to this PR: I wonder if we could help catch this in future PRs. 
For example if we used a single helper method in NullableWalker to add diagnostics, we could assert that only diagnostics that were registered as "nullable warnings" would be produced from NullableWalker.

Reply:
Yes, an assert was added.

---
In reply to: [239274559](https://github.com/dotnet/roslyn/pull/31499#discussion_r239274559) [](ancestors = 239274559)